modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/LocalizableText/LocalizableText.es.js

```diff
@@ -105,29 +106,34 @@ const LocalesDropdown = ({
 							localeId,
 						}) => (
 							<ClayDropDown.Item
-								className="autofit-row custom-dropdown-item-row"
+								className="custom-dropdown-item-row"
 								data-testid={`availableLocalesDropdown${localeId}`}
 								key={localeId}
 								onClick={(event) => {
 									onLanguageClicked({event, localeId});
 									setDropdownActive(false);
 								}}
 							>
-								<span className="autofit-col autofit-col-expand">
-									<span className="autofit-section">
-										<span className="inline-item inline-item-before">
-											<ClayIcon symbol={icon} />
-										</span>
-										{displayName}
-									</span>
-								</span>
+								<ClayLayout.ContentRow containerElement="span">
```

Comment:
@eduardoallegrini just one question does this replace the removal of `autofit-row` in `<ClayDropDown.Item />`?

Reply:
Yes, Bryce proposed a different solution in this comment https://github.com/liferay-frontend/liferay-portal/pull/56#issuecomment-649076696 but in this case I think we can use the 
dropdown-item as a wrapper for the autofit-row. It makes everything easier to read and change (I think we may have a cleaning/removing-the-unnecessary-element task in the future)

Reply:
Hmm, this is interesting. No problem!